docs/tutorials/programmatic-upgradeable-module.md

```diff
@@ -0,0 +1,266 @@
+- Publish module as a standalone (resource) account, a building block in a decentralized design where no private keys can control the resource account. The ownership (SignerCap) can be kept in another module, such as governance.
+
+The first step will go over section 1: **How to publish modules to a resource account.**
+
+## 1. How to publish modules to a resource account
+
+Before publishing the module, we want to create a resource account prior, so that the module then gets uploaded to the resource account. To do this, we run the command:
+
+1. `aptos move create-resource-account-and-publish-package --address-name <your-address> --seed <seed>`
+    1. Replace `<your-address>` with your account address.
+    2. Replace `<seed>` with a seed that only you know. This is used as a custom input to generating the resource account address.
+
+When running this command, this will **FIRST calculate** your resource account address. 
+A prompt below will show:
+
+```jsx
+Do you want to publish this package under the resource account's address 
+0x090ad1536fe5cfcb5632b3026f99f8415c55b69ce54b6f17ed8cd7edbcb5edfa? [yes/no]
+```
+
+- `0x090ad1536fe5cfcb5632b3026f99f8415c55b69ce54b6f17ed8cd7edbcb5edfa` is the resource account address calculated, note this down in your `Move.toml` file, under the `[addresses]` section.
+- Add a new entry `resource_account` and add the address generated as the value. in `Move.toml`
+
+```jsx
+[addresses]
+your_module = "0x1d3e5574728c3d0d544f5679229fac153c5f7d78b323751b41811c5ffbb21cfd"
+resource_account = "0x090ad1536fe5cfcb5632b3026f99f8415c55b69ce54b6f17ed8cd7edbcb5edfa"
+```
+
+2. Now that you have the `resource_account` address generated, terminate the operation by pressing `control + c`. 
+If you have placeholder addresses, .e.g: `_` as addresses, refer to: [Placeholder addresses](#addresses-with-empty-placeholders)
+
+
+3. You will now need to change all the module contracts addresses with the `resource_account`. For example in your module, you may have `your_module` as your account address.
+
+```move
+module your_module::user_info {
+	// Your code...
+}
+```
+
+Simply replace this with
+
+```move
+module resource_account::user_info {
+	// Your code...
+}
+```
+
+1. Now run the command again: `aptos move create-resource-account-and-publish-package --address-name <your-address> --seed <seed>`. 
+It will ask for permission to spend gas (octas) for the transaction to go through. Once confirmed, you should see a result of **success**.
+
+Congratulations, you have deployed your module onto a resource account. To verify, 
+check the [**Aptos explorer**](https://explorer.aptoslabs.com/) and search for your resource account address. 
+Click on `Modules` tab and you should see the code/module published onto the account.
+**[Example](https://explorer.aptoslabs.com/account/0x090ad1536fe5cfcb5632b3026f99f8415c55b69ce54b6f17ed8cd7edbcb5edfa/modules/code/user_info?network=devnet)**
```

Comment:
I would put a static example to look at on testnet, since devnet is restarted once a week